config.dev.json

```diff
@@ -235,6 +235,34 @@
           "decimals": 0
         }
       ]
+    },
+    {
+      "id": 5,
+      "name": "rococo-phala",
+      "type": "substrate",
+      "bridge": "",
+      "handlers": [],
+      "nativeTokenSymbol": "pha",
+      "nativeTokenFullName": "pha",
+      "nativeTokenDecimals": 18,
+      "blockConfirmations": 2,
+      "startBlock": 0,
```

Comment:
Should this be 0?
We should probably set this to the latest block currently on rococo as we don't want to index from the beginning of time.

Reply:
that makes sense, will do

Reply:
done, please check

Reply:
Is a 0 missing?
Seems low.